daisy_workflows/image_import/windows/run_startup_scripts.cmd

```diff
@@ -33,9 +32,45 @@ for /f "tokens=2 delims=:" %%a in (
   netsh interface ipv4 set dnsservers "Local Area Connection 3" static address=%%a primary
 )
 
+echo "Translate: Setting timezone" > COM1:
 tzutil /s 'UTC'
 w32tm /resync
 
+echo "Translate: Verifying network connectivity" > COM1:
```

Comment:
The logic for 32 and 64 bit versions of this script seem to be the same. Can they be extracted and reused in both scripts?

Reply:
Having this be in another files would add complexity, so I took this as "can we combine the two scripts as they're very similar" and realized that it could is we detect the file exists differences between x86 and x64.